src/foam/nanos/notification/email/ApplyBaseArgumentsEmailPropertyService.js

```diff
@@ -38,75 +38,79 @@ foam.CLASS({
         Logger logger = Loggers.logger(x, this);
 
         User user = (User) emailMessage.findUser(getX());
-        x = foam.util.Auth.sudo(x, user);
+        if ( user != null ) {
+          x = foam.util.Auth.sudo(x, user);
+          emailMessage.setSpid(user.getSpid());
+          String sendTo = (String) templateArgs.get("sendTo");
+          if ( SafetyUtil.isEmpty(sendTo) )
+            templateArgs.put("sendTo", user.getEmail());
+          templateArgs.put("locale", user.getLanguage().getCode().toString());
+        }
 
-        emailMessage.setSpid(user.getSpid());
+        // AppConfig config
+        AppConfig appConfig = (AppConfig) x.get("appConfig");
+        String url = "";
+        if ( appConfig != null ) {
+          url = appConfig.getUrl();
+          templateArgs.put("appLink", url);
+          templateArgs.put("termsAndCondLink", url + appConfig.getTermsAndCondLink());
+          templateArgs.put("termsAndCondLabel", appConfig.getTermsAndCondLabel());
+          templateArgs.put("copyright", appConfig.getCopyright());
+          templateArgs.put("privacyUrl", url + appConfig.getPrivacyUrl());
+          templateArgs.put("privacyLabel", appConfig.getPrivacy());
+        }
 
         Theme theme = (Theme) x.get("theme");
-        SupportConfig supportConfig = theme.getSupportConfig();
-        EmailConfig emailConfig = supportConfig.getEmailConfig();
-
-        // Set ReplyTo, From, DisplayName from support email config
-        if ( emailConfig != null ) {
-          // REPLY TO:
-          if ( ! SafetyUtil.isEmpty(emailConfig.getReplyTo()) ) {
-            emailMessage.setReplyTo(emailConfig.getReplyTo());
+        if ( theme != null ) {
+          if ( ! SafetyUtil.isEmpty(url) ) {
+            templateArgs.put("logo", url + "/" + theme.getLogo());
+            templateArgs.put("largeLogo", url + "/" + theme.getLargeLogo());
           }
-          // DISPLAY NAME:
-          if ( ! SafetyUtil.isEmpty(emailConfig.getDisplayName()) ) {
-            emailMessage.setDisplayName(emailConfig.getDisplayName());
+          templateArgs.put("appName", theme.getAppName());
+          // Temporary color until token support is added for email
```

Comment:
just setting 
put("theme", theme) will do, 
the templating looking for 
"theme.primary1" will find the theme object and call get Primary1 on it. 
Sorry, just noticed this is just being moved around.